README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Zulip](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://hail.zulipchat.com?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-[Hail](https://hail.is) is an open-source, general-purpose, Python-based data analysis tool with additional data types and methods for working with genomic data. Hail is used throughout academia and industry as the analytical engine for major studies, projects, and services, including the Genome Aggregation Database (gnomad.broadinstitute.org) and Neale lab mega-GWAS (nealelab.is/uk-biobank).
+[Hail](https://hail.is) is an open-source, general-purpose, Python-based data analysis tool with additional data types and methods for working with genomic data. Hail is used throughout academia and industry as the analytical engine for major studies, projects, and services, including the Genome Aggregation Database ([gnomad.broadinstitute.org](http://gnomad.broadinstitute.org)) and Neale lab mega-GWAS ([nealelab.is/uk-biobank](http://nealelab.is/uk-biobank)).
```

Comment:
The Neale lab link supports https, and I'd use that whenever possible (gnomad doesn't appear to work with https).